src/test/java/com/zipcodewilmington/looplabs/IntegerDuplicateDeleterTest.java

```diff
@@ -48,6 +48,8 @@ public void testRemoveDuplicatesExactly2() {
 
     @Test
     public void testRemoveDuplicatesExactly3() {
```

Comment:
If I recall correctly this test was broken before. In this case its fine that you edited the test, but in the future it would be preferred if you just took the changes from our the zipcoder github by fetching/pulling instead of just copy/pasting.